Perl/Splitter/Makefile.PL

```diff
@@ -18,7 +18,7 @@ my $module = "Sereal::Splitter";
  inc::Sereal::BuildTools::generate_constant_includes($module) if $in_source_repo;
 
 `rm -rf t/004_testset.t t/700_roundtrip t/data t/lib`;
-our $OPTIMIZE;
+my $optimize = inc::Sereal::BuildTools::build_optimize({ catch_violations => 0 });
```

Comment:
I can address the warnings in `srl_splitter.c` in a separate pull request and then this option is not needed any more if you like